src/util/favorites.js

```diff
@@ -133,15 +133,15 @@ const models2objects = (layer) => {
         layer.rows = layer.rows.map(cleanDimension)
     }
 
-    if (layerType === EARTH_ENGINE_LAYER) {
-        const { layerId: id, band, params, aggregationType, filter } = layer
+    if (layer === EARTH_ENGINE_LAYER) {
```

Comment:
I think you want to check `layerType` instead of `layer` (see that layer.layer was renamed on line 124)

Reply:
Fixed